MetabolicAtlas/MetabolicAtlas
# feat: redo interaction partners

**Related issue(s) and PR(s)**  
This PR closes #1144.

This is a combined effort from the whole maple team; @e0 , @inghylt , @nanjiangshu and me. 


Interaction partners now uses the [3d viewer](https://github.com/MetabolicAtlas/3d-network-viewer).

**Type of change**  


 
**List of changes made**  

- [`3dviewer`](https://github.com/MetabolicAtlas/3d-network-viewer) is used
- cytoscape is removed
- all graphs, even large ones, will be generated without warnings
- documentation is updated
- reproducible urls
- highlighting is disabled

**Screenshot of the fix**  

![2022-10-26-114521_1542x886_scrot](https://user-images.githubusercontent.com/1029190/197994032-6cd5d723-740e-48c1-852c-358c0f17141f.png)



**Testing**  

- A rebuild is needed due to new dependencies
- Try every feature of the IP page :)
- Read through the updated documentation
- Make sure all links on the web page to IP still works

**Further comments**  


**Definition of Done checklist**  
These two acceptance criteria are not fulfilled:
- Tests verifying the changes are added
- The API response is the same as before
-
